product_docs/docs/epas/16/database_administration/14_edb_clone_schema/setting_up_edb_clone_schema.mdx

```diff
@@ -9,28 +9,24 @@ In addition, it might help to modify some configuration parameters in the `postg
 
 ## Installing extensions
 
-Perform this installation on any database to be used as the source or target database by an EDB Clone Schema function.
+Install the following extensions on any database to be used as the source or target database by an EDB Clone Schema function: `postgres_fdw`, `dblink`, `edb_job_scheduler`, and `dbms_job`.
 
-1. Install the following extensions on the database: `postgres_fdw`, `dblink`, `adminpack` and `pgagent`.
-1. Ensure that pgAgent is installed before creating the `pgagent` extension. On Linux, you can use the `edb-as<xx>-pgagent` RPM package, where `<xx>` is the EDB Postgres Advanced Server version number to install pgAgent. On Windows, use StackBuilder Plus to download and install pgAgent.
-
-  Install the extensions:
-
-  ```sql
-  CREATE EXTENSION postgres_fdw SCHEMA public;
-  CREATE EXTENSION dblink SCHEMA public;
-  CREATE EXTENSION adminpack;
-  CREATE EXTENSION pgagent;
-  ```
+```sql
+CREATE EXTENSION postgres_fdw SCHEMA public;
+CREATE EXTENSION dblink SCHEMA public;
+CREATE EXTENSION adminpack;
+CREATE EXTENSION edb_job_scheduler;
+CREATE EXTENSION dbms_job;
+```
 
-  For more information about using the `CREATE EXTENSION` command, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/static/sql-createextension.html).
+For more information about using the `CREATE EXTENSION` command, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/static/sql-createextension.html).
 
 ## Modifying the configuration file
 
 Modify the `postgresql.conf` file by adding `$libdir/parallel_clone` to the `shared_preload_libraries` configuration parameter:
 
   ```ini
-  shared_preload_libraries = '$libdir/dbms_pipe,$libdir/dbms_aq,$libdir/parallel_clone'
+  shared_preload_libraries = '$libdir/dbms_pipe,$libdir/dbms_aq,$libdir/parallel_clone,$libdir/edb_job_scheduler'
```

Comment:
@gabbasb is dbms_job already included in the libraries and hence doesn't need to be loaded?